src/native/libs/System.Security.Cryptography.Native.Apple/pal_aead.swift

```diff
@@ -0,0 +1,88 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+import CryptoKit
+import Foundation
+
+@_cdecl("AppleCryptoNative_ChaCha20Poly1305Encrypt")
+public func AppleCryptoNative_ChaCha20Poly1305Encrypt(
+    keyPtr: UnsafeMutableRawPointer,
+    keyLength: Int32,
+    noncePtr: UnsafeMutableRawPointer,
+    nonceLength: Int32,
+    plaintextPtr: UnsafeMutableRawPointer,
+    plaintextLength: Int32,
+    ciphertextBuffer: UnsafeMutablePointer<UInt8>,
+    ciphertextBufferLength: Int32,
+    tagBuffer: UnsafeMutablePointer<UInt8>,
+    tagBufferLength: Int32,
+    aadPtr: UnsafeMutableRawPointer,
+    aadLength: Int32
+ ) -> Int32 {
```

Comment:
I assume that our style for .swift files would be to use Allman braces, rather than K&R.  Though I have no idea how to apply that to guard-let-else.

Reply:
Almost all swift I have ever read is written in 1TBS, including Apple's. If we want to change it to Allman that's fine, but it is very "not swifty".